GitUI/CommandsDialogs/RevisionFileTree.cs

```diff
@@ -513,14 +513,13 @@ private void FileTreeContextMenu_Opening(object sender, System.ComponentModel.Ca
             openFileWithToolStripMenuItem.Visible = isFile;
 
             toolStripSeparatorGitActions.Visible = isFile;
-            stopTrackingThisFileToolStripMenuItem.Visible = isFile;
-            stopTrackingThisFileToolStripMenuItem.Enabled = isExistingFileOrDirectory;
-            assumeUnchangedTheFileToolStripMenuItem.Visible = isFile;
-            assumeUnchangedTheFileToolStripMenuItem.Enabled = isExistingFileOrDirectory;
+            stopTrackingThisFileToolStripMenuItem.Visible = 
+                stopTrackingThisFileToolStripMenuItem.Enabled = isFile;
+            assumeUnchangedTheFileToolStripMenuItem.Visible = 
+                assumeUnchangedTheFileToolStripMenuItem.Enabled = isFile;
```

Comment:
this is wrong! The use of `assume unchanged` feature has only sense when the file has not changed. So what was done was good. Please revert.

Reply:
ok